python/setup.py

```diff
@@ -30,11 +26,18 @@ def run(self):
             "core/src/global_scheduler/global_scheduler"]
 }
 
+
+class BinaryDistribution(Distribution):
+  def has_ext_modules(foo):
+    return True
+
+
 setup(name="ray",
       version="0.1.0",
       packages=find_packages(),
       package_data=package_data,
```

Comment:
Maybe the problem is that this `package_data` line is getting applied before `build_ext` runs so there's nothing to include.

But then since the files in `package_data` have been generated, they get included the second time you run `setup.py`.